packages/dashboard-core-plugins/src/panels/DropdownFilterPanel.tsx

```diff
@@ -160,25 +225,43 @@ class DropdownFilterPanel extends Component {
     if (this.cleanup) {
       this.cleanup();
     }
-    if (valuesTable !== null) {
+    if (valuesTable != null) {
       valuesTable.close();
     }
   }
 
-  getCachedValues = memoize((rawValues, { type, name }, formatter) => {
-    if (TableUtils.isDateType(type)) {
+  dropdownFilterRef: RefObject<DropdownFilter> | null;
+
+  panelContainer: RefObject<HTMLDivElement> | null;
```

Comment:
These can never be null, the RefObject is set in the constructor:
```suggestion
  dropdownFilterRef: RefObject<DropdownFilter>;

  panelContainer: RefObject<HTMLDivElement>;
```